### DISCUSSION TITLE
DNS Subdomain - "404: NOT_FOUND"

### DISCUSSION THREAD

**Discussion by Emilprivate** [vercel/community#3594] [Help] [answered]
### Summary

Hi, I'm trying to add a custom subdomain to point to an external source, KeyAuth.

My project and domain are on Vercel.

This is the video tutorial offered by KeyAuth on how to add subdomains using Cloudflare, which works.
[Cloudflare DNS Subdomain Setup on KeyAuth](https://www.youtube.com/watch?v=iHQe4GLvgaE)

I've tried to replicate it by not using Cloudflare but directly utilizing Vercel's DNS UI:
<img width="1200" alt="image" src="https://github.com/vercel/community/assets/25754141/eac2ee98-36b8-46e3-8782-64af5c57c794">

And on KeyAuth's platform, I've added my custom subdomain and website link as per the tutorial video:
<img width="359" alt="image" src="https://github.com/vercel/community/assets/25754141/f3c3cd20-87f0-424b-b60b-bc7c68e5d692">

All of this works if I use Cloudflare, however, for some reason I can't get it to work using Vercel's DNS UI:
<img width="690" alt="image" src="https://github.com/vercel/community/assets/25754141/35095aca-436d-4c3c-b221-89303399d035">

It's only been about 30 minutes since I've added it on Vercel, so perhaps I should wait longer? I'm also getting this sometimes instead:
<img width="317" alt="image" src="https://github.com/vercel/community/assets/25754141/53240b04-5c75-47f6-90de-565b728973d7">

Any help is appreciated, thanks!

### Steps to Reproduce

- You would need both a keyauth subscription and a vercel domain to reproduce this.

**Comment by swarnava** [comment]
Hi, What is your domain name?

**Reply by Emilprivate** [reply]
www.redgoods.dev

**Reply by swarnava** [reply, marked as answer]
I can see that the CNAME is pointed correctly and served as expected from Vercel side: https://dnschecker.org/#CNAME/auth.redgoods.dev

Since we don't have control beyond this, we recommend checking with the hosting provider / application / service you are using to check configuration further.
